esm/interface/document.js

```diff
@@ -210,6 +210,25 @@ export class Document extends NonElementParentNode {
     return document;
   }
 
+  dispatchEvent(event) {
+    const dispatched = super.dispatchEvent(event);
+
+    // intentionally simplified, specs imply way more code: https://dom.spec.whatwg.org/#event-path
+    if (dispatched && event.bubbles && !event.cancelBubble) {
+      const view = this.defaultView;
+      if (view && view.dispatchEvent) {
+        const options = {
+          bubbles: event.bubbles,
+          cancelable: event.cancelable,
+          composed: event.composed,
+        };
+        // in Node 16.5 the same event can't be used for another dispatch
+        return view.dispatchEvent(new event.constructor(event.type, options));
```

Comment:
A note about bubbling as I'm still working on it:
After working on https://github.com/WebReflection/linkedom/pull/84 I noticed there was still issues there.

For a page having a `button`, a `div#parent` and a `div#container`, dispatching a click Event on the button should keep "button" as target during all bubbling while "currentTarget" would be the current element the bubbling is on.

I see the current proposition is also missing that, as the original target is lost.

Proper bubbling with correct target could be difficult to achieve while keeping the ungap/event-target or Node EventTarget extends, if I have a reasonable proposition I'll do a PR.

Reply:
I think that instead of adding dispatchEvent there, all you need could be to define `_getParent` for document that would return `this.defaultView`.
This should work as document also extends  `EventTarget`.
`dispatchEvent` is already defined there and use that `_getParent` method.

Reply:
currentTarget is always where the listener was attached, never nodes in between. target is where the initial dispatch happened.

Reply:
@Kal-Aster I opened a PR for solving the target issue but also with an update for the window propagation based on what I suggested: https://github.com/WebReflection/linkedom/pull/89
This should be more in line with existing code than adding the dispatchEvent logic to `document`, maybe you can check it to see if that works for you.

Reply:
Well, thank you for taking care of it, I couldn't find time for it these weeks but I think I couldn't make a better implementation!
It seems working for me, haven't test it but as long as the event propagates to the window, like it does, it works.
If @WebReflection agrees with it, we can also close this PR